### PR TITLE
Login

### PR DESCRIPTION
**Pull request by BJordi**
## Summary

- Fixes in login view.
- Added login services.
- Added onSubmit method in login view.

### CODE REVIEW

File: src/views/Login.vue [hunk 1]
```diff
@@ -32,22 +32,25 @@
       </div>
       <button class="primary-button">Login</button>
     </form>
+    <div class="horizontal-division"/>
```
Comment (FrankIglesias) [inline src/views/Login.vue:35]: transform this into a pug

File: src/views/Login.vue [hunk 2]
```diff
@@ -61,11 +64,26 @@ export default {
...
+    },
+    async onSubmit () {
+      if (!this.$v.$invalid) {
+        const userData = this.generateLoginObject()
+        const response = await login(userData)
+        if (response.ok) {
+          console.log(response.data.access_token)
+        } else {
+          this.formError =
+            response.data && response.data.error && response.data.error[0]
```
Comment (FrankIglesias) [inline src/views/Login.vue:81]: why do you validate the existence of at least one error?

File: src/views/SignUp.vue [hunk 3]
```diff
@@ -82,35 +82,34 @@ export default {
...
+        const userData = this.generateObject()
+        const response = await signUp(userData)
         if (response.ok) {
-          this.$router.push("/login");
+          this.$router.push('/login')
         } else {
-          debugger;
           this.formError =
-            response.data && response.data.error && response.data.error[0];
+            response.data && response.data.error && response.data.error[0]
```
Comment (FrankIglesias) [inline src/views/SignUp.vue]: same

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: src/views/Login.vue (hunk 1 above)
Comment (BJordi) [reply src/views/Login.vue:35]: Will do it in a specific PR.

File: src/views/Login.vue (hunk 2 above)
Comment (BJordi) [reply src/views/Login.vue:81]: Should I use a map method?

File: src/views/Login.vue [hunk 4]
```diff
@@ -61,11 +64,26 @@ export default {
...
+        email: this.email,
+        password: this.password
+      }
+    },
+    async onSubmit () {
+      if (!this.$v.$invalid) {
+        const userData = this.generateLoginObject()
+        const response = await login(userData)
+        if (response.ok) {
+          console.log(response.data.access_token)
```
Comment (FrankIglesias) [inline src/views/Login.vue:78]: you have to set the token in the local storage here, are you doing it in another pr?

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: src/views/Login.vue (hunk 4 above)
Comment (BJordi) [reply src/views/Login.vue:78]: This PR correspond to the `Login` card in which you just have to print the token. In the next PR, that corresponds to the card `Session management & Logout` I'll be doing what you say.

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**